packages/ui/src/services/GlobalConfigPlugin.ts

```diff
@@ -1,8 +1,9 @@
 import { ref, inject, App } from 'vue'
 
 import { getDefaultConfig } from '../components/vuestic-components/va-config/config-default'
-import { merge } from 'lodash'
+import { mergeWith } from 'lodash'
 import { colorsPresets } from './color-config/color-theme-presets'
+import { iconsPresets } from './../services/icon-config/presets'
```

Comment:
Should include only: 
* vuestic aliases (close, arrows etc);
* md default

Everything else should be passed to plugin config or set dynamically.